src/components/form.js

```diff
@@ -37,34 +37,87 @@ const AddBookForm = () => {
+                <div className="rating">
+                  <div>
+                    <div className="complited-loader flex">
+                      <div className="complited-percentage">
+                        <div className="progress blue">
+                          <span className="progress-left">
+                            <span className="progress-bar" />
+                          </span>
+                          <span className="progress-right">
+                            <span className="progress-bar" />
+                          </span>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                  <span className="vertical-divider-right separeter" />
+                  <div className="col-1">
+                    <p className="vertical-divider-right separeter" />
+                  </div>
+                  <div className="update col-3 flex" />
+                  <div>
+                    <h3 className="-Percent-Complete">64%</h3>
+                    <h4 className="Completed Text-Style-2">completed</h4>
+                  </div>
+                </div>
+                <div className="Line-2" />
+                <div>
+                  <h3 className="Current-Chapter Text-Style-7">CURRENT CHAPTER</h3>
+                  <h2 className="Current-Lesson Text-Style-4">
+                    {' '}
+                    {book.chapter}
+                    Chapter4:Human economics
+                  </h2>
+                  <div className="Rectangle-2">
+                    <h1 className="Update-progress">
+                      UPDATE PROGRESS
+                    </h1>
+                  </div>
+                </div>
               </li>
             ))}
           </ul>
         </div>
-        <form onSubmit={handleSubmit}>
-          <input type="text" name="title" placeholder="Title" value={book.title} onChange={handleChange} />
-          <input type="text" name="author" placeholder="Author" value={book.author} onChange={handleChange} />
-          <input type="text" name="category" placeholder="Category" value={book.category} onChange={handleChange} />
-          <button type="submit">Add Book</button>
+        <div className="Line" />
+        <form onSubmit={handleSubmit} className="form">
+          <span className="Add Add-Style-10">
+            ADD NEW BOOK
+          </span>
+          <div className="inputs">
+            <input type="text" name="title" placeholder="BOOK TITLE" value={book.title} onChange={handleChange} className="Leson-Panel" />
+            <input
+              type="text"
+              name="category"
+              placeholder="Category                                                ▼"
+              value={book.category}
+              onChange={handleChange}
+              className="Lesson-Panel2"
+            />
+            <button type="submit" className="Rectangle-3">Add Book</button>
+          </div>
```

Comment:
- [ ] The last one your add book form is looks like stuck to the bottom, as original degin present, please add some spaces at the bottom of it. 

<img width="1437" alt="Screenshot 2023-04-15 at 01 22 14" src="https://user-images.githubusercontent.com/69011963/232149619-48bb0569-bb0f-427b-8aa8-e8914d94425f.png">

![Screenshot 2023-04-15 at 01 36 38](https://user-images.githubusercontent.com/69011963/232149926-c7e4deb9-2da8-4a49-ae60-603453462843.png)